project/BinaryIncompatibilities.scala

```diff
@@ -10,6 +10,15 @@ object BinaryIncompatibilities {
     ProblemFilters.exclude[MissingClassProblem]("org.scalajs.ir.Trees$JSLinkingInfo$"),
     ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.scalajs.ir.Trees#*.tpe"),
     ProblemFilters.exclude[MissingClassProblem]("org.scalajs.ir.Types$NoType$"),
+
+    // !!! Breaking, OK in minor release
```

Comment:
Nit: Duplicate comment intended?

Reply:
Yes, that was intended. It's a different category of changes IMO.